### PR TITLE
refactor: rename org profile

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

**Review by l00mi — COMMENTED** [review]
Lets go the full on and actually also call it in the UI Publishing Profile ?

**Comment by tpluscode** [comment]
> Lets go the full on and actually also call it in the UI Publishing Profile ?

You mean [this](https://github.com/zazuko/cube-creator/pull/867/files#diff-3a15aec8e472f6d63c2b693d9735078b13263b2e35aabc70d15c1d5fcf8243e4R28-R29)

File: cli/lib/metadata.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ export async function injectMetadata(this: Context, jobUri: string) {
 
   const propertyShapes = new TermMap<QuadSubject, QuadObject>()
 
+  const creatorTerms = dataset.pointer.out(dcterms.creator).terms
```
Comment (martinmaillard) [inline cli/lib/metadata.ts:57]: Do if the "opendata.swiss organization" is not set in the cube metadata, observations won't have an `observedBy`. Is that ok? I thought `observedBy` was mandatory?

From the discussion last week, I thought the idea was to store the value for `observedBy` on the publishing profile?

Comment (tpluscode) [reply cli/lib/metadata.ts:57]: Ah, maybe. @l00mi ?

**Review by l00mi — APPROVED** [review]

File: cli/lib/metadata.ts [hunk 2]
```diff
@@ -159,6 +158,7 @@ export async function injectMetadata(this: Context, jobUri: string) {
     }
 
     if (quad.predicate.equals(cube.observedBy)) {
+      const creatorTerms = maintainer.pointer.out(cube.observedBy).terms
       for (const creator of creatorTerms) {
         if (creator.termType === 'NamedNode') {
```
Comment (martinmaillard) [inline cli/lib/metadata.ts:163]: Is this check needed?

Comment (tpluscode) [reply cli/lib/metadata.ts:163]: Other than satisfying the type system, technically a term can be multiple things and we do want an URI

**Comment by tpluscode** [comment]
@martinmaillard why do cypress tests fail?

**Comment by martinmaillard** [comment]
Should work now

**Pull request merged**